makefile

```diff
@@ -1022,6 +1022,19 @@ ifeq ($(OS_is_win),yes)
 $(foreach a,$(filter %_dll.$a,$(release.LIBS_A)),$(eval $(call .release.add_compat_copy,$a,$(RELEASEDIR.libia),_release_c)))
 endif
 
+# Adds symlink to the old library directory
+# Note: The `ln` command does not support `-r` on MacOS, so we
+#       `cd`to the lib directory first and then create symlink.
+ifeq ($(if $(or $(OS_is_lnx),$(OS_is_mac)),yes,),yes)
+_release_c: 
+  cd $(RELEASEDIR) && ln -sf daal dal
+endef
+
+ifeq ($(OS_is_win),yes)
+_release_c: 
+  cd $(RELEASEDIR) && mklink /D daal dal
```

Comment:
Does it work without admin access?

Reply:
yes. as we create link in some user directory and it's sufficient to have use right there